extensions/BugModal/web/bug_modal.js

```diff
@@ -213,10 +213,10 @@ $(function() {
     function expireSavedComments() {
         const AGE_THRESHOLD = 7 * 24 * 60 * 60 * 1000; // 7 days in milliseconds.
         let expiredKeys = [];
-        for (let i = 0; i < localStorage.length; i++) {
-            let key = localStorage.key(i);
+        for (let i = 0; i < window.localStorage.length; i++) {
+            let key = window.localStorage.key(i);
             if (key.match(/^bug-modal-saved-comment-/)) {
-                let value = JSON.parse(localStorage.getItem(key));
+                let value = Bugzilla.Storage.get(key);
```

Comment:
Out-of-scope nit: I don't really love the implicit JSON handling, it would be better to have a `get_json` method that handles JSON values instead.

Reply:
Not sure why this is an issue as the Bugzilla.Storage class is handling the JSON parsing and stringification behind the scenes now. So the caller will use Bugzilla.Storage.get and the value returned will be a valid JSON object or a single value. So the caller will just need to check of value is typeof object if it is not sure. Bugzilla.Storage.set looks to see if the value passed is and object and if so, it will stringify it as JSON before storing.